src/style.css

```
@@ -0,0 +1,35 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  height: 15px;
+}
+
+.stars__item {
+  display: inline-block;
+  width: 13px;
+  height: 12px;
+  margin: 0 1px;
+  background: url(images/star.svg) no-repeat;
+}
+
+.stars--1 .stars__item:nth-child(1) {
+  background: url(images/star-active.svg) no-repeat;
+}
+
+.stars--2 .stars__item:nth-child(-n+2) {
+  background: url(images/star-active.svg) no-repeat;
+}
```

селектори можна писати, через кому, щоб не повторювати одне і те саме, ось так:
```suggestion
.stars--1 .stars__item:nth-child(1),
.stars--2 .stars__item:nth-child(-n+2) {
  background: url(images/star-active.svg) no-repeat;
}
```

---

усі 5